lib/StylesheetsFileResolver.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const path = require('path');
+
+const cheerio = require('cheerio');
+const recursiveReadDir = require('recursive-readdir');
+
+const STYLESHEET_REGEXP = /{{\s*stylesheet\s*([\/a-zA-Z'"\.-]+)\s*}}/gi;
+
+class StylesheetsFileResolver {
+    constructor(themePath) {
+        this.themePath = themePath;
+    }
+
+    async get() {
+        const templatesPath = path.join(this.themePath, 'templates');
+        const files = await recursiveReadDir(templatesPath);
+        const cssFiles = [];
+        for await (const file of files) {
+            const content = await fs.promises.readFile(file, { encoding: 'utf-8' });
+            const result = content.matchAll(STYLESHEET_REGEXP);
+            if (result) {
+                for (const item of result) {
+                    // remove quotes
+                    const filePath = item[1].slice(1, -1);
+                    const fileName = this.tryToResolveCssFileLocation(filePath);
+                    if (
+                        fileName &&
+                        !this.isStyleSheetAComment(content, filePath) &&
+                        !cssFiles.includes(fileName)
+                    ) {
+                        cssFiles.push(fileName);
+                    }
+                }
+            }
+        }
+
+        return cssFiles;
+    }
+
+    isStyleSheetAComment(content, cssFilePath) {
+        const $ = cheerio.load(content);
+        const comments = $('*')
+            .contents()
+            .filter(function () {
+                return this.nodeType === 8;
```

Comment:
I'm wondering why just `8`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType#node.comment_node
I'll add a comment here for more visibility